package.json

```diff
@@ -27,28 +27,36 @@
     "url": "http://www.bentley.com"
   },
   "devDependencies": {
-    "@bentley/build-tools": "^2.10.0",
+    "@itwin/build-tools": "rc",
+    "@itwin/eslint-plugin": "rc",
     "@types/chai": "^4.2.12",
     "@types/mocha": "^5.2.7",
+    "chai": "^4.3.4",
     "cpx": "^1.5.0",
+    "eslint": "^7.11.0",
     "mocha": "^5.2.0",
     "rimraf": "^3.0.2",
-    "tslint": "^5.11.0",
-    "tslint-etc": "^1.13.6",
-    "typescript": "~3.7.4"
+    "typescript": "^4.4.1"
   },
   "dependencies": {
     "@bentley/backend-itwin-client": "^2.10.0",
-    "@bentley/bentleyjs-core": "^2.10.0",
-    "@bentley/ecschema-metadata": "^2.10.0",
-    "@bentley/frontend-authorization-client": "^2.10.0",
-    "@bentley/geometry-core": "^2.10.0",
     "@bentley/imodelhub-client": "^2.10.0",
-    "@bentley/imodeljs-backend": "^2.10.0",
-    "@bentley/imodeljs-common": "^2.10.0",
-    "@bentley/imodeljs-i18n": "^2.10.0",
     "@bentley/itwin-client": "^2.10.0",
     "@bentley/rbac-client": "^2.10.0",
-    "@bentley/telemetry-client": "^2.10.0"
+    "@bentley/telemetry-client": "^2.10.0",
```

Comment:
this should be `@itwin/core-telemetry` but I also don't think you need to take a real dependency on it.

Reply:
It looks like none of these @bentley/[client] packages are direct deps in this repo... I'll try removing all of them and see if I'm misunderstanding something

Reply:
ok I do see some required peers, will keep those

Reply:
@calebmshafer could you take a look at my pinned `@bentley/itwin-client` dep? the `rc` tag resolved to a 2.19-* dev version. If it's good you can resolve, but otherwise, what might I use?

Reply:
Do you directly use @bentley/itwin-client in the samples? I'm just about to remove that package from 3.x use so you'll want to migrate away from it.

Reply:
no it is not directly used but my local testing failed without it

Reply:
so I'll remove it and just test after the 3.x release when I need to remove `rc` versions anyway?

Reply:
Yup, sounds good to me.